tests/integration/create_cluster/ha/expected-v1alpha2.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: kops/v1alpha2
```

Comment:
So this is great - I love seeing these new tests come to life - Good job @justinsb